mygit.rb

```diff
@@ -0,0 +1,365 @@
+require 'optparse'
+require 'optparse/time'
+require 'octokit'
+require 'io/console'
+require 'logger'
+
+$force = 'False'
```

Comment:
If the option parsing and handling functions were to be inside a class , then global variables will not be necessary.